frame/mosaic/src/weights.rs

```diff
@@ -0,0 +1,68 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(clippy::unnecessary_cast)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+pub trait WeightInfo {
+	fn set_relayer() -> Weight;
+	fn rotate_relayer() -> Weight;
+	fn set_network() -> Weight;
+	fn set_budget() -> Weight;
+	fn transfer_to() -> Weight;
+	fn accept_transfer() -> Weight;
+	fn claim_stale_to() -> Weight;
+	fn timelocked_mint() -> Weight;
+	fn set_timelock_duration() -> Weight;
+	fn rescind_timelocked_mint() -> Weight;
+	fn claim_to() -> Weight;
+}
+
+// For backwards compatibility and tests
+impl WeightInfo for () {
+    fn set_relayer() -> Weight {
+        0 as Weight
```

Comment:
if would prefer even in tests that is not 0

Reply:
Not really sure what you mean by this. These weights are just there for backwards compatibility and tests? What would you like to change it to?

Reply:
I think Dzmitry would prefer them to be larger, arbitrary numbers. In case they're ever insecurely used, the damage of the vulnerability would be limited.

Reply:
:+1:

Reply:
so yeah, if we spammed, we still earn. 

also weights are used for add/remove. when store happens on behalf of users, than remove (after some conditions are met), is removed. than it is easy to ensure our tests are properly set to non-zero weights. 

also they ensure our tests are more like e2e tests. which I find useful in simulator.

Reply:
Ah I see what you mean now